### ISSUE TITLE
Tripit button not showing properly in Chrome and IE9

### ISSUE THREAD

**Issue by shijumhnsaves** [tripit/api#101]
We have a travel website where we are attempting to apply this plugin on our confirmation page after a reservation is made.  I have provided a static link we have created which you can open in both Chrome and IE9 to see the issue we are seeing.

https://www.mhnsaves.com/tripit.php?reservation_id=1  

![image 1](https://f.cloud.github.com/assets/6432370/1943546/2c193646-7fac-11e3-8d9b-201950fb8cdb.png)

**Comment by tariqislam** [comment]
Hi,

I just clicked on the link above, and I am able to see the button working on Chrome on Mac OSX. Is this bug particular to Windows, and which version of OS please?

Thanks.

Tariq

**Comment by shijumhnsaves** [comment]
Hi Tariq.

I see that bug particularly in Windows 7 version, in both browsers Chrome
and IE9.

thanks
Shiju

**Comment by tariqislam** [comment]
Thanks for the information, Shiju. I have opened up a ticket at our end to fix this issue, but it may take a while before we can resolve it due to other higher priority tasks. Please feel free to ping me for a status update in the near future.

Thanks.

Tariq

**Comment by shijumhnsaves** [comment]
Hello,

We are having issues creating the correct reservation email based on the
email standard format defined by tripit as per this link:
http://help.tripit.com/entries/23260071-Add-to-TripIt-button-standard-confirmation-email-format-for-vendors

We tried creating this exact format, and forwarding that email to
plans@tripit.com, however the reservation details are not translating into
a tripit itinerary.

The format we are applying is as follows (using filled in data), you can
also see the results of this forwarded template in my account, user id:
roshan@mhnsaves.com.  Please let us know if we are doing something
incorrect and/or let us know the proper procedure for this.  We already
attempted using the widget, which was not working properly, and so we
thought using the basic email template, we would be able to resolve this
issue.

_TripIt_\* Approved*

_Booking_\* c*_onfirmation_ _#_ : 123456

_Booking_ _date_ : Feb 10, 2014

_Booking_ _site_ _name_ : mhnsaves.com

_Booking_ _site_ _phone_ : 410-872-0307

_Booking_ _site_ _web-page_ : snl.mhnsaves.com

_Hotel_\* I*_nformation_

Hotel name : Marriott Inn and Suites Orlando

Hotel address : 2458 Summit Avenue

Hotel phone : _770-568-2845 <770-568-2845>_

Hotel confirmation # : 456789123

Guest name : Roshan Patel

Check-in date : Mar 15, 2014

Check-out date : Mar 18, 2014

Check-in time : 03:00 PM

Check-out time : 11:00 AM

Room type : 2 Queen Beds Non Smoking

Room description : Ocean View

Number of nights : 2

Number of guests : 3

Number of rooms : 1

Frequent guest # :

Cancellation remarks : Must Cancel 24 Hrs in advance

Booking rate : $100.00

Total cost : $300.00

Regards
Shiju

**Comment by tariqislam** [comment]
Hi Shiju,

Please send an email to support@tripit.com letting them know that the email is not getting parsed, with details on what you are sending (please attach the file to the email, if appropriate) and what we are neglecting to parse. Our support team with then create an issue to deal with the email parsing questions.

Thanks.

Tariq

**Comment by bryanchug** [comment]
This is the same exact problem we're also facing.

**Comment by tariqislam** [comment]
(edited)

Hi Bryanchug,

I'm not sure what issue you are referring to... the email not getting parsed correctly, or the Chrome/IE9 issue in Windows 7. Could you let me know what environment you have tested it on, and what works/doesn't? Currently we are working on a couple of bugs, hence your details would help figure out whether it is a new issue or something getting fixed.

Thanks.

Tariq

**Comment by bryanchug** [comment]
Hi Tariq,

Could you let us know instead on which environments you have tested them and confirmed that they actually work as intended, and on which websites have they been embedded on and works as intended as well?

Thanks!

**Comment by tariqislam** [comment]
Hi,

The plugin was tested on Windows 7 and Mac, across Chrome/Firefox/IE8-11. All browsers are now currently working. We do have one more set of fixes that will go out next week on Tuesday, but at least now you should be able to continue development and testing of the plugin. Please let me know immediately if that is not the case.

Thanks for your patience!

Tariq

**Comment by tariqislam** [comment]
Closing this issue since there are no more reports of errors.
